### PR TITLE
Add hiDPI support for Windows

### LINKED ISSUE #168: Add hiDPI support for Windows build
- [x] I have read the specified guidelines for issues

**Describe the bug**
Main window is low DPI and blurry.

**To Reproduce**
1) Install Tenacity or pre-fork 
2) Run
3) Window is blurry

**Expected behavior**
UI should be scaled to match DPI of monitor

**Additional information (please complete the following information):**
 - OS:  Windows 10
 - Version: Latest (3.0.2)

**Additional context**
It is an old upstream problem, [this forum post](https://forum.audacityteam.org/viewtopic.php?t=102856) has the solution which can be applied as a post-install script in the existing Windows Inno Setup.

### PR DESCRIPTION
**Pull request by fossdd**
Resolves: https://github.com/tenacityteam/tenacity/issues/168

Hopefully, adds hiDPI support for Windows. We still need to verify that with a real windows machine.

- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

### CODE REVIEW

**Comment by emabrey** [comment]
image
image

**Comment by emabrey** [comment]
I tested it on windows and it built, but it won't start.

**Comment by AnotherFoxGuy** [comment]
Also tested on windows, same issue

**Comment by nbsp** [comment]
This PR builds and launches fine for me on a Windows VM.

**Comment by fossdd** [comment]
Okay, awesome. Is HiDPI enabled? @SFR-git

**Comment by nbsp** [comment]
> Okay, awesome. Is HiDPI enabled? @SFR-git

I'm not sure? I've not really checked for it, I just wanted to see if it runs. The app looks fine both with and without the patch.

*Disclaimer: I've not used Windows in... a while.*

**Comment by emabrey** [comment]
My monitor is 141.69 pixels/in (calculated it myself and verified it with an online calculator for PPI), which is considered hiDPI I believe. I built and started this build succesfully, and it seems much less blurry to me:

image

More info on my monitor, if it helps:

`Diagonal -> sqrt((19.4 cm*19.4 cm)  + (34.4 cm *34.4 cm)) = 39.5 cm`
`PPC -> 2202 pixels / 39.5 cm = 55.7 ppc`
`PPI -> 2202 pixels / 15.6 in =  141.2 ppi`

image
image
image

**Comment by nbsp** [comment]
> I built and started this build succesfully, and it seems much less blurry to me:

Could you review this PR, then?

**Review by emabrey — DISMISSED** [review]

**Comment by emabrey** [comment]
> 
> 
> > I built and started this build succesfully, and it seems much less blurry to me:
> 
> Could you review this PR, then?

I thought I did. My bad.

**Comment by nbsp** [comment]
Unless someone else with a HiDPI monitor can test, I think we can merge this PR.

**Comment by fugidev** [comment]
The CI build does not launch on my system ("side by side configuration invalid")

**Comment by fossdd** [comment]
> The CI build does not launch on my system ("side by side configuration invalid")

Which windows version do you use?

**Comment by fugidev** [comment]
> Which windows version do you use?

Windows 10 21H1 Build 19043.1110

File: win/Tenacity.exe.manifest [hunk 1]
```diff
@@ -6,8 +6,12 @@
...
       type="win32"
   />
-  <description>Tenacity Cross-Platform Sound Editor</description>
-  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v3">
+  <asmv3:application>
+    <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">
+      <dpiAwareness>PerMonitorV2, unaware</dpiAwareness>
+    </asmv3:windowsSettings>
+  </asmv3:application>
+  <description>Tenacity Cross-Platform Sound Editor</description>">
```
Comment (Fox2Code) [inline win/Tenacity.exe.manifest]: the `">` at the end look like a typo

Comment (fugidev) [reply win/Tenacity.exe.manifest]:
```suggestion
  <description>Tenacity Cross-Platform Sound Editor</description>
  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v3">
```
After changing this, it built successfully.

Comment (fossdd) [reply win/Tenacity.exe.manifest]: Okay, looks like ca8a67b16f2810d466e07a02c3c015f6c12474ff did the error. @FugiMuffi Please retry with f57559194

**Comment by nbsp** [comment]
Please squash on merge

File: win/Tenacity.exe.manifest (hunk 1 above)
Comment (fugidev) [reply win/Tenacity.exe.manifest]: This line was removed by ca8a67b16f2810d466e07a02c3c015f6c12474ff @fossdd. Without it that's not valid xml.
```xml
<trustInfo xmlns="urn:schemas-microsoft-com:asm.v3">
```

File: win/Tenacity.exe.manifest [hunk 2]
```diff
@@ -6,8 +6,12 @@
       name="Tenacity.Tenacity.Tenacity"
       type="win32"
   />
+  <asmv3:application>
+    <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">
+      <dpiAwareness>PerMonitorV2, unaware</dpiAwareness>
+    </asmv3:windowsSettings>
+  </asmv3:application>
   <description>Tenacity Cross-Platform Sound Editor</description>
-  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v3">
```
Comment (fugidev) [inline win/Tenacity.exe.manifest:10]: This line was removed by accident by ca8a67b16f2810d466e07a02c3c015f6c12474ff.

**Review by fugidev — CHANGES_REQUESTED** [review]

**Review by fugidev — APPROVED** [review]

**Review by nbsp — APPROVED** [review]

**Pull request merged**